From 2cebf0889a2a51f34366e93818dbf087bf4d4913 Mon Sep 17 00:00:00 2001
From: Naoto Yamaguchi <naoto.yamaguchi@aisin.co.jp>
Date: Sat, 10 Sep 2022 21:21:53 +0900
Subject: [PATCH] Fix meson error in 0.61.x

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1bdb9c7b32..b50c4a7802 100644
--- a/meson.build
+++ b/meson.build
@@ -38,7 +38,7 @@ conf.set_quoted('RELATIVE_SOURCE_PATH', relative_source_path)
 want_ossfuzz = get_option('oss-fuzz')
 want_libfuzzer = get_option('llvm-fuzz')
 want_fuzzbuzz = get_option('fuzzbuzz')
-if want_ossfuzz + want_libfuzzer + want_fuzzbuzz > 1
+if (want_ossfuzz and want_libfuzzer) or (want_ossfuzz and want_fuzzbuzz) or (want_libfuzzer and want_fuzzbuzz)
         error('only one of oss-fuzz, llvm-fuzz or fuzzbuzz can be specified')
 endif
 
-- 
2.25.1

